.github/workflows/ci.yml

```diff
@@ -3,7 +3,7 @@ on:
   push:
     branches:
       - master
-  pull_request_target:
+  pull_request:
```

Comment:
@gaecoli I did this change to match changes on master. Not sure why it shows up in diff, but it's fine.